aries-site/src/pages/templates/drill-down-navigation.mdx

```diff
@@ -1,3 +1,50 @@
-import { ComingSoon } from '../../components';
+import { DiagramPreview, Example } from '../../layouts';
+import { DrillDownCardExample, DrillDownDataTableExample, DrillDownListExample } from '../../examples';
 
-<ComingSoon />
\ No newline at end of file
+## When to Use:
+- When users need to focus on one task at a time.
+- To give information overview with the ability to drill down into specific information.
+- When screen real estate is limited.
+- When designing for many levels of functionality.
+- To divide meaningful content into sections.
+
+## Guidelines:
+- When the user is moving forward step-by-step to access further information.
+- When going from your Account > Dashboard > Service.
+- To allow users to dive deeper into related content.
+- Can be used as an individual element or in groups of related content.
+- Each element should have it’s own single content.
+
+## Drill Down Navigation Examples:
+
+### Card Example
+A visual representation of interactive and organized content.
+
+<Example caption="A diagram illustrating drill down navigation within Cards." plain pad="small" previewWidth="large" height={{ min: 'small' }}>
```

Comment:
I think the caption should be differentiated from the alt text. In this case, they are the same and a screen reader user will hear the same content repeatedly.

I think the text is good for the alt text, but the caption could serve more as a narrative which accentuates and accompanies the illustration, explaining what the reader should takeaway from the illustration (kind of like a nature film where the narrator is explaining the live action visuals).

e.g. "Illustration showing how cards may be used to navigate from a central "hub" page to "spoke" pages of related content and/or more granular detail."

Reply:
Thanks for this feedback I will work on the narrative for the other examples